spec/unit/remote_system_spec.rb

```diff
@@ -19,11 +19,13 @@
 
 describe RemoteSystem do
   let(:remote_system) { RemoteSystem.new("remotehost") }
+  let(:ssh_basic_options) { [ "-o", "ControlMaster=auto", "-o", "ControlPath=~/.ssh/" \
```

Comment:
Block body expression is on the same line as the block start.<br>Space inside square brackets detected.